Quick start updates, Example updates, and Updates to SBO topics for modular-docs template and content type attributes alignment (#41809)
openshift/openshift-docs: modules/sbo-data-model.adoc

// Module included in the following assemblies:
//
// * /applications/connecting_applications_to_services/exposing-binding-data-from-a-service.adoc

:_content-type: CONCEPT
[id="sbo-data-model_{context}"]
= Data model

[role="_abstract"]

> This section explains the data model used in the annotations and OLM descriptors.

Reviewer: Generally, RH docs avoid self-reference phrases like "This section explains..."
Does the following suggestion accurately reflect the purpose of this topic?
```suggestion
The data model used in the annotations and OLM descriptors follow specific conventions.
```

Reviewer: IBM SG: Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes".
Instead, focus on the purpose or value of the content, and include words that users are likely to search
for. For example, in a short description in an online help topic, provide users with enough information to
determine whether the topic meets their needs. You can use self-referential text in learning modules
and tutorials.

`<NAME>`:: Specifies the name under which the binding value is to be exposed. You can exclude it only when the `objectType` parameter is set to `Secret` or `ConfigMap`.
`<VALUE>`:: Specifies the constant value exposed when no `path` is set.

> The data model is the same for custom resource definition (CRD) or custom resource (CR) annotations and Operator Lifecycle Manager (OLM) descriptors, but the syntax differs.

Reviewer: ```suggestion
Although the data model is the same for custom resource definitions (CRD), custom resource (CR) annotations, and Operator Lifecycle Manager (OLM) descriptors, the syntax for each one differs.
```
